lib/Wikia/src/Service/User/Permissions/PermissionsService.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Wikia\Service\User\Permissions;
+
+interface PermissionsService {
```

Comment:
I can see you didn't reply to one of my suggestions to introduce a "bound PermissionsService" using:
`$this->permissionsService()->user( $someUserObject )->_aMethodFromThisInterface_()`

Could you share your point of view on this topic?